docs/src/apireference.md

```diff
@@ -304,3 +307,4 @@ Bridges.LogDetBridge
 Bridges.SOCtoPSDCBridge
 Bridges.RSOCtoPSDCBridge
 ```
+For each bridge defined in this package, a corresponding bridge optimizer is available with the same name but "Bridge" suffix, e.g. `SplitInterval` is an `SingleBridgeOptimizer` for the `SplitIntervalBridge`.
```

Comment:
but _without the_ "Bridge" suffix?

Also, I'm not sure what our policy is on `e.g.`, but it either needs to be "suffix, e.g., `SplitInterval`" or "suffix (e.g. `SplitInterval`"